src/community/infix-long-scale.ts

```diff
@@ -0,0 +1,31 @@
+import { AbstractInfixNotation } from "./infix-abstract";
+import Decimal from "break_infinity.js";
+import { toSubscript, abbreviate } from "../utils";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixLongScaleNotation extends AbstractInfixNotation {
+  public get name(): string {
+    return "Infix long scale";
+  }
+  
+  protected groupDigits = 6;
```

Comment:
Codacy found an issue: [expected member-variable-declaration: 'groupDigits' to have a typedef](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6218257)